.gitignore

```diff
@@ -1,3 +1,4 @@
 scalalib/src/
 target/
 *.dot
+image0.ppm
```

Comment:
I don't think that we should have this one in gitignore as it's really demo-specific.